internal/rpc/rpcserver/server.go

```diff
@@ -1926,6 +1937,59 @@ func (s *walletServer) Spender(ctx context.Context, req *pb.SpenderRequest) (*pb
 	return resp, nil
 }
 
+func (s *walletServer) GetCFilters(req *pb.GetCFiltersRequest, server pb.WalletService_GetCFiltersServer) error {
+	var startBlock, endBlock *wallet.BlockIdentifier
+	if req.StartingBlockHash != nil && req.StartingBlockHeight != 0 {
```

Comment:
minor nit but I find a switch easier to read here than the if else if else if:

```
switch {
case req.StartingBlockHash != nil && req.StartingBlockHeight != 0:
    // error
case req.StartingBlockHash != nil:
    // set startBlock
case req.StartingBlockHeight != nil:
    // set startBlock
}
```

Reply:
Fixed in https://github.com/decred/dcrwallet/pull/1705/commits/ffbf7612d0002c2c995328413589808e7a7a320f by refactoring all `{Starting,Ending}Block{Hash,Height}` decoding in a single function.